src/main/java/org/folio/innreach/config/JpaAuditingConfig.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.innreach.config;
+
+import java.util.Optional;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.data.domain.AuditorAware;
+import org.springframework.data.jpa.repository.config.EnableJpaAuditing;
+
+@Configuration
+@EnableJpaAuditing(auditorAwareRef = "auditorAware")
```

Comment:
I'd add modifyOnCreate=false to avoid setting modification date upon create

Reply:
Ok, will add this